usmqe/api/graphiteapi/graphiteapi.py

```diff
@@ -89,17 +94,22 @@ def compare_data_mean(
                         expected_number_of_datapoints, len(graphite_data)))
             LOGGER.debug("mean of data from `{}` in Graphite: {}".format(
                 target, graphite_data_mean))
-            graphite_data_mean_sum += graphite_data_mean
+            if operation == 'sum' or graphite_data_mean_all == 0:
```

Comment:
As far as I understand, here the function relies heavily on the fact that there are no negative numbers in the data otherwise the subtraction could produce wrong results. While it seems OK for our product, I think it should be at least clearly documented.